src/server/ServerOptions.ts

```diff
@@ -0,0 +1,15 @@
+export interface ServerOptions {
+    ServerName: string;
+    Server_IP: string;
```

Comment:
Refrain from mixing naming conventions, use `CamelCase`not `under_score`

Reply:
k son